### ISSUE TITLE
dependencies to weewxwd?

### ISSUE THREAD

**Issue by Bierchermuesli** [gjr80/weewx-highcharts#2]
not 100% researched and not longer testet -just for info... It looks like hfw has some dependencies on weewxwd, I don't reall see a need in them...

`cheetahgenerator: Generated 14 files for report StandardReport in 37.90 seconds
imagegenerator: Generated 12 images for StandardReport in 5.10 seconds
copygenerator: copied 9 files to /var/www/weewx
reportengine: Caught unrecoverable exception in generator weewx.cheetahgenerator.CheetahGenerator
         ****  Unknown data binding ''wd_binding''
         ****  Traceback (most recent call last):
         ****    File "/usr/share/weewx/weewx/reportengine.py", line 239, <snip>
`

so I installed `wee_extension --install=./weewxwd-1.0.3.tar.gz`:
`
cheetahgenerator: Generated 14 files for report StandardReport in 37.49 seconds
imagegenerator: Generated 12 images for StandardReport in 5.13 seconds
copygenerator: copied 9 files to /var/www/weewx
reportengine: Caught unrecoverable exception in generator weewx.cheetahgenerator.Che
etahGenerator
        ****  unsupported operand type(s) for /: 'NoneType' and 'float'
        ****  Traceback (most recent call last):
         ****    File "/usr/share/weewx/weewx/reportengine.py", line 239, in run
<snip>
`

happens on weewx  out of the box v3.7.1 and 3.6.2 (dep's)  with mysql backend

I wasn't successful to generate json - I ended up with a simple simple php script.

**Comment by gjr80** [comment]
Thanks. I initially put this extension together to simplify any re-installations I may have to do and thought I might as well put it out there for others to use as they see fit. Clearly I did not test very thoroughly!

In any case I have removed the hard coded weeWX-WD bindings. The extension now looks for weeWX-WD bindings for `appTemp` and `insolation`. If they exist they are used, if not the extension looks for a user specified `apptemp_binding` and `insolation_binding` in the `[[Highcharts]]` stanza under `[StdReport]` in `weewx.conf`. If no bindings are found then the `appTemp` and `insolation` data is omitted from the week and year JSON files.

This proved a little more complex than just amending the report templates as the javascript needed work as well to handle that case where there is no `appTemp` and/or `insolation` data - it looked odd having no data display but still have 'Apparent Temperature' or 'Theoretical Max Solar radiation' on the legend.

I have released v0.2.0 that appears to be well behaved on a VM running simulator with no weeWX-WD.

I will update the wiki User's Guide in due course.
